index.rst

```diff
@@ -41,40 +41,46 @@ with version control systems.
 File Format
 ===========
 
-EditorConfig files use an INI format that is compatible with the format used
-by `Python configparser Library`_. In an EditorConfig file (usually named
-``.editorconfig``), all beginning whitespace in each line is ignored. Each
-line must be one of the following:
-
-- Empty Line: An empty string.
-- Comment: A line starting with a ``;`` or a ``#``.
-- Section Title: A line that starts with a ``[``, and that ends with a ``]``.
-- Assignment: A line separated by an `=` into two parts.
-
-Any line that is not one of the above is invalid. "Inline comment" in
-appearance, i.e., a line that has a ``;`` or a ``#`` in the middle, is
-undefined. For convenience, we also define the following terminologies:
-
-- Section Name: The string between the beginning ``[`` and the ending ``]`` in
-  a Section Title.
-- Section: The lines starting from a Section Title until the beginning of the
-  next Section Title or end of file.
-- Property: The part before the first `=` in an Assignment.
-- Value: The part after the first `=` in an Assignment.
-
-In EditorConfig files, the Section Names are filepath globs, similar to the
-format accepted by gitignore. ``[`` and ``]`` are allowed in the Section
-Names. Forward slashes (``/``) are used as path separators and Backslashes
-(``\\``) are not allowed as path separators (even on Windows). Comments should
-always go on individual lines. EditorConfig files should be UTF-8 encoded, with
-either CRLF or LF line separators.
+EditorConfig uses a strict EditorConfig-INI file format. In an EditorConfig
```

Comment:
That was @florianb's suggestion, because now the INI file format specification is locked down with the additional clarification we need. It no longer leans on the 3rd party Python ConfigParser specification. It stands on its own, strictly.

Why do you think we don't need this? What is your definition of strict that wouldn't apply here?

Reply:
In future iterations this also should reference to `editorconfig/ini` to make it even more clear.

Reply:
This is only a language suggestion: I think this is a bit redundant and pretty much every concept is strict in this document. It is also "strictly" UTF-8 encoded, keys are strictly to the left of `=`, etc.

Reply:
Removed strict in 605c0c4